src/services/item/plugins/embeddedLink/schemas.ts

```diff
@@ -15,25 +18,27 @@ export const createSchema = S.object()
   .prop('extra', embeddedLinkItemExtraCreate)
   .required(['type', 'extra']);
 
-// equivalent in a static JSON Schema
-// {
-//   type: 'object',
-//   properties: {
-//     type: { const: 'embeddedLink' },
-//     extra: {
-//       type: 'object',
-//       additionalProperties: false,
-//       properties: {
-//         embeddedLink: {
-//           type: 'object',
-//           additionalProperties: false,
-//           properties: {
-//             url: { type: 'string', format: 'url' }
-//           },
-//           required: ['url']
-//         }
-//       },
-//       required: ['embeddedLink']
-//     }
-//   }
-// }
+// schema defining the allowed properties for the link item
+export const updateExtraSchema = S.object()
+  .additionalProperties(false)
+  .prop(
+    ItemType.LINK,
+    // only allow to set url in extra on create
```

Comment:
I'm not sure to understand where this constraint is reflected in this schema. It is a description of the next line or an indication that we should allow to set the url in extra on create only ? I think it is the same for the createExtraSchema.